io_ogre/ui/export.py

```diff
@@ -163,18 +163,53 @@ def execute(self, context):
                 kw[ name[3:] ] = getattr(self,name)
         config.update(**kw)
 
-        print ("_"*80)
+        print ("_" * 80)
+        
         target_path, target_file_name = os.path.split(os.path.abspath(self.filepath))
         target_file_name = clean_object_name(target_file_name)
         target_file_name_no_ext = os.path.splitext(target_file_name)[0]
 
-        logger.debug(" Target_path: %s" % target_path)
-        logger.debug(" Target_file_name: %s" % target_file_name)
-        logger.debug(" Target_file_name_no_ext: %s" % target_file_name_no_ext)
+        file_handler = None
+        
+        # Add a file handler to all Logger instances
+        if config.get('EXPORT_ENABLE_LOGGING') == True:
+            log_file = ("%s\\blender2ogre.log" % target_path)
+            logger.info("Writing log file to: %s" % log_file)
+
+            file_handler = logging.FileHandler(filename=log_file, mode='w', encoding='utf-8', delay=False)
+            file_formatter = logging.Formatter(fmt='%(asctime)s [%(levelname)5s] %(message)s', datefmt='%Y-%m-%d %H:%M:%S')
+            if config.get('SHOW_LOG_NAME') == True:
```

Comment:
```suggestion
            if SHOW_LOG_NAME:
```

Reply:
You think it should be an internal option?